pkg/image/resolve.go

```diff
@@ -130,7 +130,8 @@ func setRequirementImages(osType OSType, imagesSet map[string]map[string]struct{
 	case Linux:
 		addSourceToImage(imagesSet, settings.ShellImage.Get(), coreLabel)
 		addSourceToImage(imagesSet, settings.MachineProvisionImage.Get(), coreLabel)
-		addSourceToImage(imagesSet, "rancher/mirrored-library-busybox:1.34.1", coreLabel)
+		addSourceToImage(imagesSet, "registry.suse.com/bci/bci-busybox:15.4.11.2", coreLabel)
+		addSourceToImage(imagesSet, "registry.suse.com/bci/bci-micro:15.4.14.3", coreLabel)
```

Comment:
We could use sp3 instead, but sp4 is stable now. So we should instead advance the other images up to sp4.